src/Java.md

```diff
@@ -120,7 +120,14 @@ It would be appreciated if there are any Java VTK experts who could convert any
 
 ## Working with 3D Data
 
+| Example Name | Classes Demonstrated | Description | Image |
+| -------------- | ---------------------- | ------------- | ------- |
+[Bottle](/Java/Modelling/Bottel) | vtkRotationalExtrusionFilter | sweeps polygonal data creating "skirt" from free edges and lines, and lines from vertices
```

Comment:
Problems:
- File name is wrong, should be "Bottle" not "Bottel"
- "sweeps"should be "Sweeps"

Reply:
Have changed as suggested.